consensus/dbft/payload.go

```diff
@@ -19,8 +19,17 @@ type (
 		BlockIndex     uint64
 		ValidatorIndex byte
 		ViewNumber     byte
+		msgPayload     interface{}
+	}
 
-		payload io.Serializable
+	// "external" message rlp encoding type.
+	messageExt struct {
```

Comment:
Let's rename it to have common code style:
```suggestion
	// messageAux is an auxiliary structure for message RLP encoding.
	messageAux struct {
```